Code/lib/corner/corner.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CORNER_H
+#define CORNER_H
+
+typedef enum{
+  LEFT,
+  RIGHT,
+  FORWARD,
+  BRAKE,
+}turn_direction;
+
+
+class CornerMerchantry
+{
+private:
+    bool wall_detected(float, float);
+    float *set_threshold_right;
+    float *set_threshold_front;
+public:
+    CornerMerchantry(float*, float*);
+    ~CornerMerchantry();
```

Comment:
Why did you include this one yet you have implemented it?

Reply:
solved